elephant/asset.py

```diff
@@ -444,12 +443,13 @@ def _jsf_uniform_orderstat_3d(u, alpha, n, verbose=False):
     lists = [range(j, n + 1) for j in range(d, 0, -1)]
     it_todo = np.prod([n + 1 - j for j in range(d, 0, -1)])
 
+    log_1 = np.log(1.)
     # Compute the log of the integral's coefficient
-    logK = np.sum(np.log(np.arange(1, n + 1))) - n * np.log(1 - alpha)
-    # Add to the 3D matrix u a bottom layer equal to alpha and a
+    logK = np.sum(np.log(np.arange(1, n + 1))) - n * log_1
```

Comment:
log(1) is zero
if it's not a mistake, drop a comment, explaining why the log power simplifies to `1`. Maybe, because it should be `1-alpha` and `alpha` is zero?
` - n*0` is not necessary then, up to you to leave it

Reply:
Yes, it is correct, we just set `alpha=0` everywhere now. I kept it for the same reason we have the other `log(1)` in the same function; to try and make it more clear how the formula was derived. But I guess you just proved that it only raises more questions here, so I removed it. The formula with `alpha=0` will probably be more familiar to potential readers anyway.

Reply:
Is there a specific formula when `alpha=0` in the paper? If yes, then the comment of not subtracting `n log(1-alpha)` is not needed.